src/Config.php

```diff
@@ -3750,4 +3750,111 @@ final public static function generateUuid($type)
         self::setConfigurationValues('core', [$type . '_uuid' => $uuid]);
         return $uuid;
     }
+
+    /**
+     * Get email and name to be used as "From" in our email notifications
+     *
+     * @param int  $entities_id
+     * @param bool $allow_response
+     *
+     * @return array [email => sender address, name => sender name]
+     */
+    public static function getAdminEmailSender(
+        int $entities_id,
+        bool $allow_response
+    ): array {
+        if ($allow_response) {
+            return self::getAdminEmail($entities_id);
+        } else {
+            return self::getNoReplyEmail();
+        }
+    }
+
+    /**
+     * Try to find a valid email to use as a sender for notifications
+     * Form first to last, it will look for :
+     * - A valid "from_email" address defined in the global configuration
+     * - A valid "admin_email" address defined in the current entity configuration
+     * - A valid "admin_email" address defined in the global configuration
+     *
+     * @param int $entities_id Current entity
+     *
+     * @return array [email => sender address, name => sender name]
+     */
+    public static function getAdminEmail(int $entities_id): array
```

Comment:
In `NotificationTarget::getSender()`, the global value is evaluated first, and entity config is used as fallback, but in `NotificationTarget::getReplyTo()`, the opposite is done.
IMHO, as long as we provide and `$entities_id` parameter, we should rely on entity config, and use global config only as a fallback.

Also, maybe we should allow asking for the global config directly, when the `null` value is passed in `$entities_id` param.
```suggestion
    public static function getAdminEmail(?int $entities_id = null): array
```